actors/util/adt/permission.go

```diff
@@ -0,0 +1,13 @@
+package adt
+
+// MutationPermission is the mutation permission on a state field
+type MutationPermission int
+
+const (
+	// Invalid means NO permission
+	InvalidPermission MutationPermission = iota
+	// ReadOnlyPermission allows reading but not mutating the field
+	ReadOnlyPermission
+	// WritePermission allows mutating the field
+	WritePermission
```

Comment:
These duplicate the same thing in market actor state. I would understand either centralizing them, or uniformly un-exporting them, but not having some private and some re-usable versions.